.eslintrc.yml

```diff
@@ -6,6 +6,7 @@ extends: 'eslint:recommended'
 globals:
```

Comment:
I'm curious if eslint runs on `.mjs`

Reply:
yup, just needs to be globbed in the script that runs it